js/components/seismic-eruptions-map.js

```diff
@@ -258,7 +263,14 @@ export default class SeismicEruptionsMap extends PureComponent {
           }
         </Map>
         {!config.showUserInterface &&
-          <div className='scale-markers'>{zoom > 3 ? `Scale: ${scaleWidth}km x ${scaleHeight}km   Zoom level: ${zoom}` : ''}</div>
+          <div className='scale-markers'>
+            {zoom > 3 &&
```

Comment:
Why do we have this check and why this particular value (3)?

Reply:
Leaflet api claims that calculating distances will return meters, but when the distances are significantly large (somewhere over 100k km) it seems to return values in km (I tested with logging the raw values before applying the various m => km, rounding, and `toLocaleString`). At zoom level 3 it was hit or miss as to whether the numbers appeared within a reasonable range, and it depended heavily on screen size. Added to the fact that map projection has role to play that becomes particularly noticeable at lower levels of zoom, I took the simplest approach of sticking to 4 and above.